### ISSUE TITLE
Is the esp32 module compatible with a generic PS3 controller?

### ISSUE THREAD

**Issue by BobEsponja12** [darthcloud/BlueRetro#469]
My question is if the esp32 module is compatible with the ps2 mode with a generic PS3 control

**Comment by vicsidious** [comment]
Yes it is! To pair a PS3 controller, you just have a few more steps to go through.
Here's a little tutorial to help you get going.
https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#ps3
